config/full.toml

```diff
@@ -67,10 +67,16 @@ throttle = "10kB,10MB" # limit and burst per second; only opendal backends; Defa
 
 # Snapshot-filter options: These options apply to all commands that use snapshot filters
 [snapshot-filter]
-filter-hosts = ["host2", "host2"] # Default: []
+filter-hosts = ["host1", "host2"] # Default: []
 filter-labels = ["label1", "label2"] # Default: []
 filter-tags = ["tag1,tag2", "tag3"] # Default: []
+filter-tags-exact = ["tag1,tag2", "tag2"] # Default: []
 filter-paths = ["path1", "path2,path3"] # Default: []
+filter-paths-exact = ["path1", "path2,path3"] # Default: []
+filter-after = "2024-01-01" # Default: not set
+filter-before = "2024-02-05 12:15" # Default: not set
+filter-size = "200MiB" # Default: not set
+filter-size-added = "1 MB..10MB" # Default: not set
```

Comment:
```suggestion
filter-size-added = "1MB..10MB" # Default: not set
```

Reply:
Would it work with a whitespace? Would be good to add that to the test cases then

Reply:
It should also work with whitspaces..